tests-arquillian/pom.xml

```diff
@@ -145,6 +145,24 @@
             <artifactId>wildfly-arquillian-common</artifactId>
         </dependency>
 
+        <!-- Jakarta JAX/JWS are needed in jdk8+ -->
```

Comment:
@starksm64 Why are these needed in JDK 8? Are they also required in newer JDKs?
Also, should a similar fix be added to 3.1 branch?